docs/tutorial/fle.txt

```diff
@@ -0,0 +1,164 @@
+======================
+Client Side Encryption
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Cliend Side Encryption allows administrators and developers to encrypt specific
+data fields in addition to other MongoDB encryption features.
+
+
+Automatic encryption and decryption
+-----------------------------------
+
+.. note::
+
+   Auto encryption is an enterprise only feature.
+
+The following example assumes the key and schema have already been created in
```

Comment:
Should there be an example of creating the key and schema? I assume this differs from the second example where we use a `schemaMap`.

Reply:
Added key and schema creation using `$jsonValidator` to the example. I would also suggest that the example in the Java documentation is improved the same way, as that's where this example was copied from.